rcsb/db/mongo/PdbxLoader.py

```diff
@@ -1018,8 +1019,7 @@ def __validateAndFix(self, databaseName, collectionName, dList, docIdL, schemaLe
             except Exception as e:
                 logger.exception("Validation updating processing error %s", str(e))
             #
-            if updL:
-                rList.append(dD)
+            rList.append(dD)
```

Comment:
A few questions/comments about `__validateAndFix `:

1. Is your intention to change the behavior here? I was surprised to learn that `if []` is True in Python (unlike, say, Javascript). So the original line is actually checking the length of updL.
2. If that was the intention to remove the length check, then `rList` is not really needed since now it's just appending all the items already in `dList`. The caller just assigns the return to the variable it passed in anyway.
4. The doc comments for this method are all generic. It might be worth doing a quick update to clarify what's going on.

Reply:
Edited my comment, I meant to say `if []` is false in python.

Reply:
Sounds like a good plan.